### DISCUSSION TITLE
为什么我的绳网主页一直在刷新啊淦

### DISCUSSION THREAD

**Discussion by RoadtoSanctuary** [share121/inter-knot#1020] [其他]
![Screenshot_20240807_134251](https://github.com/user-attachments/assets/b33133c6-b02c-4156-8c42-63dc6dbe7c70)

**Comment by isyuah** [comment]
确定不是因为图片懒加载而导致帖子高度变化，进而导致的的帖子移动么，这个是一次性请求全部的数据的(前100条)，理论上不可能一直刷新
